otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1304,6 +1304,18 @@ orgs.newOrg('eclipse-tractusx') {
         },
       ],
     },
+    orgs.newRepo('emergingtechnologies') {
+      delete_branch_on_merge: true,
+      is_template: true,
```

Comment:
is the intentional? This would make the repo a template repo so others can create repos based on that.
Just wanted to double check as this is quite unusual.

Reply:
Oh nope just a new repo... will update my Pr

Reply:
0705d08 removed template option, thanks for the hint @netomi